cue/data/gen.cue

```diff
@@ -22,7 +22,7 @@ Family: scuemata.#Family & {
                 // Theme of dashboard.
                 style: *"light" | "dark"
                 // Timezone of dashboard,
-                timezone?: *"browser" | "utc"
+                timezone?: *"browser" | "utc" | ""
```

Comment:
seems optional and "" should be equivalent use cases?
this could be 
timezone  : *"browser" | "utc" | "" or
timezone?: *"browser" | "utc"